kickstarts/base.ks.erb

```diff
@@ -69,30 +69,18 @@ unset DEBUG
 # See: https://github.com/ManageIQ/manageiq/pull/6751
 export RAILS_USE_MEMORY_STORE="true"
 
-# ensure /usr/local/bin is on the path (for ruby and virtualenv)
+# ensure /usr/local/bin is on the path (for virtualenv)
 export PATH=$PATH:/usr/local/bin
 
 <%= render_partial "post/firewalld" %>
 
-<%= render_partial "post/source_setup" %>
-
 <%= render_partial "post/repos" %>
 
 <%= render_partial "post/ovirt_ansible" %>
 
 <%= render_partial "post/python_modules" %>
 
-<%= render_partial "post/ruby_install" %>
-
-<%= render_partial "post/bundler" %>
-
-# needs to run *after* post/bundler
-<%= render_partial "post/ui_compile" %>
-
-<%= render_partial "post/rake_tasks" %>
-
-# appliance_root="/opt/manageiq/manageiq-appliance" -- in post/source_setup partial
-$appliance_root/manageiq-setup.sh
+/opt/manageiq/manageiq-appliance/manageiq-setup.sh
```

Comment:
Does this needs to be productizable? (We can do it in a follow up PR)

Reply:
Updated.